commands.js

```diff
@@ -1044,16 +1044,48 @@ exports.commands = {
 			move = Commands.randommove.call(this, "2,viable", self, dummyRoom);
 		}
 
+		//Code for generating a random name based on types generated
+		//Start by accessing the array from the capnamelist.js file in the data folder.
+		var foundName = false;
+		var t1Name = '';
+		var type1array = Data.Capnames[type[0]];
+		var type2array = Data.Capnames[type[1]];
+		while(!foundName){
+            t1Name = Tools.sample(type1array)[0]
```

Comment:
The reverse indent here is weird and I don't understand it. The whitespace as a whole isn't in the same style as the rest of the code for the bot, and I'd appreciate it if you could try to match the style of the rest of the code. Namely putting spaces before tokens.